src/main/kotlin/org/jitsi/jibri/selenium/status_checks/EmptyCallStatusCheck.kt

```diff
@@ -44,6 +46,13 @@ class EmptyCallStatusCheck(
     private fun CallPage.isCallEmpty() = getNumParticipants() <= 1
 
     companion object {
-        val DEFAULT_CALL_EMPTY_TIMEOUT: Duration = Duration.ofSeconds(30)
+        private val defaultCallEmptyTimeout: Duration by config {
+            "JibriConfig::defaultCallEmptyTimeout" {
+                Duration.ofSeconds(Config.legacyConfigSource.defaultCallEmptyTimeout!!)
+            }
+            "jibri.default-call-empty-timeout".from(Config.configSource).convertFrom<Long>(Duration::ofSeconds)
+            "default" { Duration.ofSeconds(30) }
```

Comment:
Instead of putting defaults here, you'll want to add them to `reference.conf`

Reply:
@bbaldino I moved the defaults to `reference.conf`, however the configuration from the `config.conf` does not seem to override it. I am sure I'm doing something wrong ...

Reply:
Couple things to check:

1. Are you passing the path to `config.conf` to Jibri when starting up? (`-Dconfig.file=/path/to/config.conf`)
1. Did you check that the property names and paths in `config.conf` match `reference.conf` and both of them match what's in the code?

Reply:
Yes on both points, I verified that.